src/cpp/client/secure_credentials.cc

```diff
@@ -104,6 +104,15 @@ std::unique_ptr<Credentials> ServiceAccountCredentials(
       json_key.c_str(), scope.c_str(), lifetime));
 }
 
+// Builds JWT credentials.
+std::unique_ptr<Credentials> JWTCredentials(
+    const grpc::string &json_key, std::chrono::seconds token_lifetime) {
+  gpr_timespec lifetime = gpr_time_from_seconds(
+      token_lifetime.count() > 0 ? token_lifetime.count() : 0);
```

Comment:
I think that it's better to fail early. It's easier to debug. What do you think?